concepts/admin.py

```diff
@@ -8,17 +11,31 @@ def resolve(modeladmin, request, queryset):
         authorities.resolve(type(obj), obj)
 resolve.verbose_name = 'resolve selected concepts'
 
+def merge_concepts(self, request, queryset):
+    return render_to_response('admin/merge_concepts.html', {'concepts': queryset})
+    add_tag.short_description = "Merge Concepts"
+
 class ConceptAdmin(admin.ModelAdmin):
     model = Concept
-    search_fields = ('label',)
-    list_display = ('label', 'resolved', 'concept_state', 'typed',)
-    actions = (resolve,)
-    list_filter=('concept_state', 'typed',)
+    search_fields = (   'label',    )
+    list_display = (    'label', 'resolved','concept_state'  )
+    actions = ( resolve,merge_concepts,    )
```

Comment:
The whitespace is all messed up here. There should be no whitespace between the parens and the quotes, and there should be a space after each comma.